Example/Tests/TransactionReceipt/EthTransactionReceiptIT.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Created by Timofey on 3/20/18.
+// Copyright (c) 2018 CocoaPods. All rights reserved.
+//
+
+import Nimble
+import Quick
+@testable import Web3Swift
+
+final class EthTransactionReceiptIT: XCTestCase {
+
+    func testExistingTransactionReceipt() {
+        expect{
+            try EthTransactionReceipt(
+                network: InfuraNetwork(
```

Comment:
@biboran Let's use MainnetTestingNetwork instead of InfuraNetwork with metamask params

Reply:
@rockfridrich sure